tests/test_sac_paper_matching.py

```diff
@@ -0,0 +1,448 @@
+                'submission_reviewer_assignment': 'Automatic',
+                'How did you hear about us?': 'ML conferences',
+                'Expected Submissions': '50',
+                'use_recruitment_template': 'Yes',
+                'hide_fields': ['pdf']
+            },
+            forum=request_form.forum,
+            invitation='openreview.net/Support/-/Request{}/Revision'.format(request_form.number),
+            readers=['{}/Program_Chairs'.format('TSACM/2024/Conference'), 'openreview.net/Support'],
+            referent=request_form.forum,
+            replyto=request_form.forum,
+            signatures=['~Program_MatchingChair1'],
+            writers=[]
+        ))
+
+        helpers.await_queue()
+
+    def test_setup_matching(self, client, openreview_client, helpers):
+
+        pc_client = openreview.Client(username='pc@matching.org', password=helpers.strong_password)
+        pc_client_v2=openreview.api.OpenReviewClient(username='pc@matching.org', password=helpers.strong_password)
+
+        request_form=pc_client.get_notes(invitation='openreview.net/Support/-/Request_Form', sort='tmdate')[0]
+        venue = openreview.helpers.get_conference(client, request_form.forum, 'openreview.net/Support', setup=False)
+
+        submissions = venue.get_submissions(sort='number:asc')
+
+        ## setup matching with no SACs
+        matching_setup_note = pc_client.post_note(openreview.Note(
+            content={
+                'title': 'Paper Matching Setup',
+                'matching_group': 'TSACM/2024/Conference/Senior_Area_Chairs',
+                'compute_conflicts': 'Default',
+                'compute_affinity_scores': 'No'
+
+            },
+            forum=request_form.id,
+            replyto=request_form.id,
+            invitation=f'openreview.net/Support/-/Request{request_form.number}/Paper_Matching_Setup',
+            readers=['TSACM/2024/Conference/Program_Chairs', 'openreview.net/Support'],
+            signatures=['~Program_MatchingChair1'],
+            writers=[]
+        ))
+        helpers.await_queue()
+
+        comment_invitation_id = f'openreview.net/Support/-/Request{request_form.number}/Paper_Matching_Setup_Status'
+        matching_status = client.get_notes(invitation=comment_invitation_id, replyto=matching_setup_note.id, forum=request_form.forum, sort='tmdate')[0]
+        assert matching_status
+        assert 'Could not compute affinity scores and conflicts since there are no Senior Area Chairs. You can use the \'Recruitment\' button to recruit Senior Area Chairs.' in matching_status.content['error']
+
+        openreview_client.add_members_to_group(
+            'TSACM/2024/Conference/Senior_Area_Chairs', 
+            ['~SAC_MatchingOne1', '~SAC_MatchingTwo1', '~SAC_MatchingThree1']
+        )
+
+        # enable bidding
+        now = datetime.datetime.utcnow()
+        due_date = now + datetime.timedelta(days=1)
+        bid_stage_note = pc_client.post_note(openreview.Note(
```

Comment:
is the bidding interface showing the papers sorted by affinity scores?

maybe we should upload the scores before starting the bidding to check this.

Reply:
I impersonate the SAC and it looks like they are being sorted by affinity scores. 

Should I add this to the test as well?

Reply:
no, that's fine. thanks for checking.